test_conformance/basic/test_vector_creation.cpp

```diff
@@ -67,102 +60,123 @@ int create_kernel(ExplicitType type, int output_size, char *program, int *number
     int total_program_length = 0;
     int aborted_due_to_size = 0;
 
-    if (skip_to_result < 0)
-        skip_to_result = 0;
+    if (skip_to_result < 0) skip_to_result = 0;
 
     // The line of code for the vector creation
     char line[1024];
-    // Keep track of what size vector we are using in each position so we can iterate through all fo them
+    // Keep track of what size vector we are using in each position so we can
+    // iterate through all fo them
     int pos[DEPTH];
     int max_size = output_size;
     if (DEBUG > 1) log_info("max_size: %d\n", max_size);
 
     program[0] = '\0';
-    sprintf(program, "%s\n__kernel void test_vector_creation(__global %s *src, __global %s%s *result) {\n",
-            type == kDouble ? "#pragma OPENCL EXTENSION cl_khr_fp64 : enable" : "",
-            get_explicit_type_name(type), get_explicit_type_name(type), ( number_of_sizes == 3 ) ? "" : size_names[number_of_sizes-1]);
+    sprintf(program,
+            "%s\n__kernel void test_vector_creation(__global %s *src, __global "
+            "%s%s *result) {\n",
+            extension, get_explicit_type_name(type),
+            get_explicit_type_name(type),
+            (number_of_sizes == 3) ? "" : size_names[number_of_sizes - 1]);
     total_program_length += (int)strlen(program);
 
-    char storePrefix[ 128 ], storeSuffix[ 128 ];
+    char storePrefix[128], storeSuffix[128];
 
     // Start out trying sizes 1,1,1,1,1...
```

Comment:
Can we update this comment to describe what we're doing with `inital_no_sizes` to omit the already tested vload vector widths

Reply:
Done